lib/src/widgets/send_message_widget.dart

```diff
@@ -0,0 +1,258 @@
+                            ),
+                            padding: const EdgeInsets.fromLTRB(9, 9, 9, 30),
+                            child: Container(
+                              margin: const EdgeInsets.only(bottom: 2),
+                              padding: const EdgeInsets.symmetric(
+                                vertical: 4,
+                                horizontal: 6,
+                              ),
+                              decoration: BoxDecoration(
+                                color: widget
+                                        .sendMessageConfig?.replyDialogColor ??
+                                    Colors.grey.shade200,
+                                borderRadius: BorderRadius.circular(12),
+                              ),
+                              child: Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                mainAxisSize: MainAxisSize.min,
+                                children: [
+                                  Row(
+                                    mainAxisAlignment:
+                                        MainAxisAlignment.spaceBetween,
+                                    children: [
+                                      Text(
+                                        "${PackageStrings.replyTo} $replyTo",
+                                        style: TextStyle(
+                                          color: widget.sendMessageConfig
+                                                  ?.replyTitleColor ??
+                                              Colors.deepPurple,
+                                          fontWeight: FontWeight.bold,
+                                          letterSpacing: 0.25,
+                                        ),
+                                      ),
+                                      IconButton(
+                                        constraints: const BoxConstraints(),
+                                        padding: EdgeInsets.zero,
+                                        icon: Icon(
+                                          Icons.close,
+                                          color: widget.sendMessageConfig
+                                                  ?.closeIconColor ??
+                                              Colors.black,
+                                          size: 16,
+                                        ),
+                                        onPressed: _onCloseTap,
+                                      ),
+                                    ],
+                                  ),
+                                  replyMessage.messageType.isImage
+                                      ? Row(
+                                          children: [
+                                            Icon(
+                                              Icons.photo,
+                                              size: 20,
```

Comment:
can this be user provided?

Reply:
No, we added as default, yet user can add parent widget of this widget